### PR TITLE
Lab6

### PR DESCRIPTION
**Pull request by none-word**

### CODE REVIEW

**Comment by Maxkoz777** [comment]
Great job!

**Comment by afonyaa** [comment]
Well done!

File: ansible/playbooks/app_java/main.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+- name: App_java
+  hosts: 51.250.89.99
```
Comment (AKurmazov) [inline ansible/playbooks/app_java/main.yml:3]: nit: You could probably just specify `yacloud`, and it would fetch the external IP every time it's changed

**Comment by AKurmazov** [comment]
Approved 👍

**Pull request closed without merging**